GoogleForCreators/web-stories-wp
# PPC: 2.0 Feature Flag & POC

## Context
Part of PPC refactor and redesign:
https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories---Sprint-1.7%2B?node-id=8378%3A1112

## Summary
Adds the initial toggle and popup for the new PPC component behind the new feature flag `enableChecklistCompanion`
<img width="1440" alt="Screen Shot 2021-06-18 at 2 14 19 PM" src="https://user-images.githubusercontent.com/35983235/122607215-dcbfb380-d03f-11eb-9689-75e9f1d86bd0.png">
<img width="771" alt="Screen Shot 2021-06-18 at 2 14 05 PM" src="https://user-images.githubusercontent.com/35983235/122607223-de897700-d03f-11eb-82f4-2d904fa4b194.png">


## Relevant Technical Choices
Pulled a lot of stuff out of the help center because the popups are so similar in the designs. I feel like once the PPC settles down a bit we should pull the common components between this, the help center, and the keyboard shortcuts popup into the design system. 

Tried not to make this a bigger change than it needed to be right now tho because so much is in flux with the PPC our focus should be focused on that and we should let some of the dust settle there before we start focusing on abstractions between the popups imo.


## To-do



## User-facing changes
NA (should all be behind a feature flag)


## Testing Instructions
Go to experiments and enable `enableChecklistCompanion`, then go to the editor and see that the new toggle and popup appear for checklist. there's really nothing in the checklist popup right now except some place holder text, which is intentional and soon to come in a later PR


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7965
